CHECKOUT_code

```diff
@@ -36,7 +36,7 @@ release="main"
 fv3_release=$release
 phy_release=$release
 fms_release="2023.02"
-drivers_release="2023.02"
+drivers_release=$release
```

Comment:
FMS and drivers should be kept in sync as much as possible.  There may be some updates in drivers::main that require a specific version of fms.